### PR TITLE
Move debug file to {fmt}

### PR DESCRIPTION
**Pull request by lefticus**
Pull request overview
---------------------

Move dbg from GIO to {fmt}

Expected changes:

Some debug tables were previously hitting format size limits and outputting `**********`, with the new formatting routines, the number is output instead

Note: dbg is not tested at all with the current tests.

**However, I did run diffs for both the DebuggingOutput and Developer options for EnergyPlus locally**

### CODE REVIEW

**Comment by lefticus** [comment]
@kbenne I cannot reproduce the Mac failure on Linux. Is this an expected/known/transient failure?

**Comment by Myoldmopar** [comment]
The Mac does have a couple test cases that fail periodically.  Almost always guaranteed to be something not being reset in the clear_state calls.  Based on the CI results from prior to your last push and after your last push I would not worry about it.  I don't think it's anything you did.

File: src/EnergyPlus/DataGlobals.cc [hunk 1]
```diff
@@ -307,7 +306,7 @@ namespace DataGlobals {
         OutputFiles::getSingleton().eso.close();
         OutputStandardError = 0;
         StdOutputRecordCount = 0;
-        OutputFileDebug = 0;
+        OutputFiles::getSingleton().debug.close();
```
Comment (Myoldmopar) [inline src/EnergyPlus/DataGlobals.cc:309]: :+1: :+1: :+1:

File: src/EnergyPlus/DataRuntimeLanguage.cc [hunk 2]
```diff
@@ -108,7 +108,7 @@ namespace DataRuntimeLanguage {
     int const OperatorGreaterThan(12);    // >
     int const OperatorRaiseToPower(13);   // ^
     int const OperatorLogicalAND(14);     // &&
-    int const OperatiorLogicalOR(15);     // ||
+    int const OperatorLogicalOR(15);     // ||
```
Comment (Myoldmopar) [inline src/EnergyPlus/DataRuntimeLanguage.cc:111]: I dunno.  I kinda liked Operatior.  And it's been in there for 11 years.  OK fine.

File: src/EnergyPlus/RuntimeLanguageProcessor.cc [hunk 3]
```diff
@@ -1360,314 +1368,65 @@ namespace RuntimeLanguageProcessor {
                     Token(NumTokens).Type = TokenOperator;
                 }
 
-                // First check for two character operators:  == <> <= >=
+                // parse an operator if found,
```
Comment (Myoldmopar) [inline src/EnergyPlus/RuntimeLanguageProcessor.cc:1371]: So now you're just doing some extra stuff :upside_down_face: :+1:

**Review by Myoldmopar — APPROVED** [review]
No problems here, I think this is good to go in.  Another file down!

**Comment by Myoldmopar** [comment]
Pulled in latest develop locally just to make sure no other develop branch movement caused issues.  It built just fine and ran tests just fine.  This is good to go in.  Thanks @lefticus for taking down another file.

File: src/EnergyPlus/RuntimeLanguageProcessor.cc (hunk 3 above)
Comment (lefticus) [reply src/EnergyPlus/RuntimeLanguageProcessor.cc:1371]: Yeah, sorry I couldn't bring myself to change the same code in 50 places when one simple function meant I only had to change it in one place.

**Pull request merged**